knotx-adapters/knotx-adapter-common/src/main/java/com/cognifide/knotx/adapter/common/placeholders/UriTransformer.java

```diff
@@ -60,4 +63,12 @@ private static String getPlaceholderValue(ClientRequest request, String placehol
         .findFirst()
         .orElse("");
   }
+
+  private static String encodeValue(String value) {
+    try {
+      return URLEncoder.encode(value, "UTF-8").replace("+", "%20").replace("%2F", "/");
+    } catch (UnsupportedEncodingException var3) {
+      throw new UnsupportedCharsetException("UTF-8");
```

Comment:
Maybe we should at least log `var3` ? (btw. please rename it to some standard `e`).